### PR TITLE
Show average star rating

### LINKED ISSUE #5436: Show average star rating
screen shot 2018-06-27 at 11 01 58 am

Change the "Overall rating" copy on the detail page to "[number] star rating". Show to one decimal point ("4.3" for example).

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #5436 

Before:

screenshot 2018-07-25 17 27 27

After: 

screenshot 2018-07-25 17 26 46

### CODE REVIEW

**Comment by rebmullin** [comment]
for some reason I am having issue with my image tool. here are the links:

https://www.evernote.com/shard/s300/sh/bee1d83b-1238-45bc-8d60-e059c8dbf473/5e179a888104c92b

https://www.evernote.com/l/ASw1Z3tUTHJJ07D5PB1RwTnIlBsJ-zzjFrs

**Comment by rebmullin** [comment]
@bobsilverberg, I also see some flow check issues. So you should check that too :)

**Comment by bobsilverberg** [comment]
Thanks for the reviews @rebmullin and @willdurand. I've addressed the concerns, but I also noticed, when looking at the issue, that there's an inconsistency in the requirements, so I've asked a couple of questions there. I'm going to mark this as blocked pending responses from @pwalm.

**Comment by bobsilverberg** [comment]
Ok, this has been updated according to the comment at https://github.com/mozilla/addons-frontend/issues/5436#issuecomment-408123161.

File: tests/unit/amo/components/TestAddonMeta.js [hunk 1]
```diff
@@ -114,6 +118,42 @@ describe(__filename, () => {
...
+
+      const averageMeta = root.find(MetadataCard).prop('metadata')[2];
+      expect(averageMeta.title).toEqual('Not rated yet');
+    });
+
+    it('renders the average rating', () => {
+      const average = 2.34;
+      const root = renderRatings({ average });
+
+      const averageMeta = root.find(MetadataCard).prop('metadata')[2];
```
Comment (willdurand) [inline tests/unit/amo/components/TestAddonMeta.js]: Getting the data with an index looks a bit fragile, maybe we could create a helper function to `getAverageMeta({ root })`?

**Pull request merged**